mapping.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Do have a look at the Slurm commands. I have a doubt about the ntasks commands. number of tasks ( What should it be in my case?
Am i taking up too many cpus or memory ? Need your help :(

Reply:
1. ntasks should be one in you case. It's used for parallelizing within a single script.
2. Yes, your code is not parallelized at this level, so there is no point using 64 cpus, just use two. Memory is fine, I would suggest increasing it to 40G depending on the size of your individual chromosome files.